Add a presubmit to verify no new JS dependencies

mindhog opened the pull request:
Verify that we have a known set of javascript dependencies.  This guards
against the inadvertent introduction of a new dependency with a disallowed
license.

TESTED: Added a new package to packages.json, observed presubmit failure.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/google/nomulus/1334)
<!-- Reviewable:end -->

gbrodman reviewed (approved):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1334)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mindhog)



<!-- Sent from Reviewable.io -->

mindhog commented:
Note: kokoro appears to be running the presubmits with a version of python older than 3.6 (which is the version we're trying to request in our kokoro configs).  I've replaced the f-strings in this PR (a feature added in 3.6) and also added a print statement to display the python version running the script.

gbrodman commented:
ugh 

python version is 3.5.2 (default, Jan 26 2021, 13:30:48)

gbrodman reviewed (approved):
Why replace the f-strings? They're the more recent version of python formatting. 

> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1334#-:-MkIIqei3uJjlzGBrTvO:bs62s8l)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mindhog)



<!-- Sent from Reviewable.io -->

mindhog reviewed (commented):
Well, because they don't work on python 3.5.2.  (we use the old-style formatting elsewhere in the file, as well... we punted on this previously)

The alternative would be to try to fix the version of python being used in this PR? (I'd prefer to take that up separately)

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1334#-:-MkIjUAVBWLWrPBUkcms:b-pus8fu)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mindhog)



<!-- Sent from Reviewable.io -->

gbrodman reviewed (approved):
yeah I made that comment before our discussion -- I guess it just didn't get published until now? Weird 

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1334#-:-MkIohsE4gau1qg7FOXR:b-f2shfu)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mindhog)



<!-- Sent from Reviewable.io -->

mindhog reviewed (commented):
haha, ok.  Thought there was some misunderstanding....

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1334#-:-MkIpGhVE-0c9_7JpDZo:b-m7wf54)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mindhog)



<!-- Sent from Reviewable.io -->
